release-notes/opensearch-security.release-notes-2.0.0.0-rc1.md

```diff
@@ -0,0 +1,24 @@
+## 2022-04-20 Version 2.0.0.0
+
+Compatible with OpenSearch 2.0.0-rc1
+
+### Enhancements
+
+* Incremented version to 2.0-rc1. ([#1764](https://github.com/opensearch-project/security/pull/#1764))
+* Renames securityconfig folder to config in bundle step and makes relevant changes ([#1749](https://github.com/opensearch-project/security/pull/#1749))
+* Add signal/wait model for TestAuditlogImpl ([#1758](https://github.com/opensearch-project/security/pull/#1758))
+* Remove TransportClient auth/auth ([#1701](https://github.com/opensearch-project/security/pull/#1701))
```

Comment:
This is a `Maintenance`.

Reply:
We do not have a firm definition for these terms, I'd like to consider as many items as possible to be enhancements as they improve how our build functions and codebase operates.

For your consideration `Renames securityconfig folder to config in bundle step and makes relevant changes ([#1749](https://github.com/opensearch-project/security/pull/#1749))` was done to improve user experience when browsing the plugin folder hierarchy.